themes/default/theme.lua

```diff
@@ -1,6 +1,7 @@
 ---------------------------
 -- Default awesome theme --
 ---------------------------
+-- luacheck: no check comment
```

Comment:
Correction needed at least:
`--luacheck: no max line length`

Besides that, I don't see a code to ignore line lengths we can set in `.luacheckrc` for the file.